packages/@dcl/sdk/src/index.ts

```diff
@@ -18,6 +19,12 @@ export async function onUpdate(deltaTime: number) {
  * Function that is called before the first update and after the evaluation of the code.
  */
 export async function onStart() {
+  const compositeProvider = await createContentFetchCompositeProvider()
```

Comment:
done in https://github.com/decentraland/js-sdk-toolchain/pull/447/commits/a1fb2ac9f5d1279ed38883cbeea212b74923d226
see how some bundle-size (like cube.ts) decrease drastically, and the `composite.ts` snapshot is the only which imports the `~system/Scene` because it needs it for fetch content mappings
TLDR, now to add this feature, instead of `export * from '@dcl/sdk'` in your scene, you put `export * from '@dcl/sdk/with-composite'`